cfp.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<meta
+			name="keywords"
+			content="OTC, Our Tech Community, OTC CatchUp, Catch Up, CatchUp, Talks, Technical Community"
+		/>
+		<meta
+			name="description"
+			content="Fill details for giving talks on OTC CatchUp."
+		/>
+		<meta name="author" content="Harsh Kapadia" />
+		<meta property="og:site_name" content="OTC Talks" />
+		<meta property="og:title" content="OTC Talks CFP" />
+		<meta property="og:type" content="website" />
+		<meta
+			property="og:description"
+			content="Fill details for giving talks on OTC CatchUp."
+		/>
+        <meta property="og:url" content="https://talks.ourtech.community/cfp" />
+		<meta
+			property="og:image"
+			content="./static/img/cfp.png"
+		/>
+		<meta property="og:image:width" content="1200" />
+		<meta property="og:image:height" content="600" />
+		<meta property="twitter:card" content="summary_large_image" />
+		<meta
+			property="twitter:image"
+			content="./static/img/cfp.png"
+		/>
+
+		<script>
+			if (screen.width < "760")
+				window.location = "https://forms.gle/x7n2JFPRRGKiPc9W7";
```

Comment:
Sometimes browsers block redirects like this, always inform the user they are being redirected and give them a link to where.

It might be worth having the redirecting notice be controlled by a css rule so it is not effected by javascript.

Reply:
Hmm, it makes sense.